code/modules/mob/living/carbon/xenomorph/charge_crush.dm

```diff
@@ -391,11 +391,20 @@
 			. = (CHARGE_SPEED(charge_datum) * 240)
 			charge_datum.speed_down(2)
 			return
-
+		
 	for(var/m in buckled_mobs)
 		unbuckle_mob(m)
 	return (CHARGE_SPEED(charge_datum) * 20) //Damage to inflict.
 
+/obj/structure/razorwire/pre_crush_act(mob/living/carbon/xenomorph/charger, datum/action/xeno_action/ready_charge/charge_datum)
+	if(CHECK_BITFIELD(resistance_flags, INDESTRUCTIBLE) || charger.is_charging < CHARGE_ON)
+		charge_datum.do_stop_momentum()
+		return PRECRUSH_STOPPED
+	if(anchored)
+		. = (CHARGE_SPEED(charge_datum) * 50) //Damage to inflict. 3 rams, 2 times taking damage.
```

Comment:
>//Damage to inflict. 3 rams, 2 times taking damage.

This comment doesn't make much sense.
What does 3 rams mean here?

Reply:
3 charges against the razorwire, actually charging it as a crusher, you take damage the 2 first times, and then on the third it just is broken.

Reply:
Ah i see what you mean now